eksrollup/cli.py

```diff
@@ -231,7 +231,19 @@ def update_asgs(asgs, cluster_name):
                         raise Exception('Instance is failing to terminate. Cancelling out.')
 
                     between_nodes_wait = app_config['BETWEEN_NODES_WAIT']
-                    if between_nodes_wait != 0:
+                    between_nodes_wait_pod_regex = app_config['BETWEEN_NODES_WAIT_POD_REGEX']
+                    if between_nodes_wait_pod_regex:
+                        try:
+                            pods_ready = pods_in_ready_state(between_nodes_wait_pod_regex)
+                            while not pods_ready:
```

Comment:
You also need to bail out at some point because this will loop forever if the pods are _never_ ready.

Reply:
That is correct, IDK when... Suggestions?

Reply:
have a `retries` variable with a value of 20 let's say. That would induce a delay of 600 seconds **top**, unless pods are ready earlier. If the retries has reached 0, get out of loop and  exit the script...smth like this

Reply:
What about pods like ElasticSearch needing more time?

Reply:
¯\_(ツ)_/¯ - retries = 60 (1800 seconds)

Reply:
i don't know if this is overthinking... possible to maybe have it prompt user input of "wait for response or bail out" after running out of retries?  that may be silly, but it would let us be like "no no this is elasticsearch it's just gonna take a while... keep trying buddy..." 😂

Reply:
I am also willing to wait indefinitely... Since we would be informed about the pod we wait on, we would be able to consider abort anyway at any time...